.github/actions/run-docker/action.yml

```diff
@@ -37,11 +37,6 @@ runs:
         DOCKER_SERVICES: ${{ inputs.services }}
         HOST_UID: ${{ steps.id.outputs.id }}
       run: |
```

Comment:
This check resulted in false negatives frequently as characters in inputs.run can break the string and inputs.run is multiline which can cause weird interpretation by bash. Not worth the effort of maintaining.